pkg/controller/bootstrap/bootstrapsigner.go

```diff
@@ -181,16 +181,18 @@ func (e *Signer) serviceConfigMapQueue(ctx context.Context) {
 	}
 	defer e.syncQueue.Done(key)
 
-	e.signConfigMap(ctx)
+	err := e.signConfigMap(ctx)
+	e.handleErr(err, key)
+
 }
 
 // signConfigMap computes the signatures on our latest cached objects and writes
 // back if necessary.
-func (e *Signer) signConfigMap(ctx context.Context) {
-	origCM := e.getConfigMap()
+func (e *Signer) signConfigMap(ctx context.Context) error {
+	origCM, err := e.getConfigMap()
```

Comment:
why to call `return e.configMapLister.ConfigMaps(e.configMapNamespace).Get(e.configMapName)` directly here?